pod.go

```diff
@@ -756,6 +756,20 @@ func (p *Pod) stopSetStates() error {
 	return nil
 }
 
+// stopShims stops all remaining shims corresponfing to not started/stopped
+// containers.
+func (p *Pod) stopShims() error {
+	for _, c := range p.containers {
+		if err := stopShim(c.process.Pid); err != nil {
+			return err
+		}
+	}
+
+	virtLog.Infof("Shim(s) stopped")
```

Comment:
Rather than this message (or maybe in combination) how about adding a `stopping shim pid %d` log call before the `stopShim()` call?